client/src/connection/rest/SASContentAdapter.ts

```diff
@@ -738,6 +742,36 @@ class SASContentAdapter implements ContentAdapter {
 
     return `${getResourceIdFromItem(myFavoritesFolder)}/members/${favoriteId}`;
   }
-}
 
+  private async checkForUnsavedFilesInFolder(
+    item: ContentItem,
+    action: "delete" | "recycle",
+  ): Promise<boolean> {
+    if (!isContainer(item)) {
+      return true;
+    }
+    const children = await this.getChildItems(item);
```

Comment:
Looks like this can only get one level of children. VS Code native file explorer can track across multiple levels.
I'm concerned that it will take too long to traverse all children.